Source/WebCore/loader/PolicyContainer.h

```diff
@@ -39,11 +40,19 @@ struct PolicyContainer {
     CrossOriginEmbedderPolicy crossOriginEmbedderPolicy;
     CrossOriginOpenerPolicy crossOriginOpenerPolicy;
     ReferrerPolicy referrerPolicy = ReferrerPolicy::Default;
+#if ENABLE(LOCAL_NETWORK_ACCESS)
+    IPAddressSpace ipAddressSpace = IPAddressSpace::Public;
+#endif
 
     friend bool operator==(const PolicyContainer&, const PolicyContainer&) = default;
 
+#if ENABLE(LOCAL_NETWORK_ACCESS)
+    PolicyContainer isolatedCopy() const & { return { contentSecurityPolicyResponseHeaders.isolatedCopy(), crossOriginEmbedderPolicy.isolatedCopy(), crossOriginOpenerPolicy.isolatedCopy(), referrerPolicy, ipAddressSpace }; }
+    PolicyContainer isolatedCopy() && { return { WTFMove(contentSecurityPolicyResponseHeaders).isolatedCopy(), WTFMove(crossOriginEmbedderPolicy).isolatedCopy(), WTFMove(crossOriginOpenerPolicy).isolatedCopy(), referrerPolicy, ipAddressSpace }; }
```

Comment:
This is a lot of duplicate code.  Can we instead just protect one member with #if ENABLE?